src/platform_impl/macos/view.rs

```diff
@@ -337,8 +337,14 @@ extern "C" fn frame_did_change(this: &Object, _sel: Sel, _event: id) {
             userData:ptr::null_mut::<c_void>()
             assumeInside:NO
         ];
-
         state.tracking_rect = Some(tracking_rect);
+
+        let logical_size = LogicalSize::new(rect.size.width as f64, rect.size.height as f64);
```

Comment:
Could you add an explanation comment why resize is done here, so it'll be easier to understand the motivation behind it?

Reply:
done